### PR TITLE
Add audio cue and toast on user join settings

### PR DESCRIPTION
**Pull request by capilkey**
This PR adds a new setting that plays a sound when a new user joins. I'm no sound guy so I just copied the chat indication, but it should really be replaced with something unique.

### CODE REVIEW

**Comment by capilkey** [comment]
PR updated to include an additional option to show a toast message when a new use joins the session.

File: bigbluebutton-html5/imports/startup/client/base.jsx [hunk 1]
```diff
@@ -311,6 +311,31 @@ const BaseContainer = withTracker(() => {
     },
   });
 
+  Users.find({}, { fields: { validated: 1, name: 1 } }).observe({
```
Comment (antobinary) [inline bigbluebutton-html5/imports/startup/client/base.jsx]: Please consider adding `if (Settings.application.userJoinAudioAlerts || Settings.application.userJoinPushAlerts)` so we don't observe unless we need to

File: bigbluebutton-html5/private/config/settings.yml [hunk 2]
```diff
@@ -30,6 +30,8 @@ public:
         animations: true
         chatAudioAlerts: false
         chatPushAlerts: false
+        userJoinAudioAlerts: true
+        userJoinPushAlerts: true
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml]: Please set the default value to false

File: bigbluebutton-html5/imports/startup/client/base.jsx (hunk 1 above)
Comment (antobinary) [reply bigbluebutton-html5/imports/startup/client/base.jsx]: I would also consider alerting only moderators. I don't see value in notifying viewers about users joining

Comment (antobinary) [reply bigbluebutton-html5/imports/startup/client/base.jsx]: Technically I think it's possible for this code to notify me for my arrival. Could filter by userId as well

File: bigbluebutton-html5/private/config/settings.yml (hunk 2 above)
Comment (capilkey) [reply bigbluebutton-html5/private/config/settings.yml]: Default are changed to false

File: bigbluebutton-html5/imports/startup/client/base.jsx (hunk 1 above)
Comment (capilkey) [reply bigbluebutton-html5/imports/startup/client/base.jsx]: The observer will now only exist if one of the settings is turned on. I also added an exclusion to not alert if the changed user is the local user.

**Pull request merged**